scripts/build-testing-app.sh

```diff
@@ -34,14 +35,7 @@ cordova plugin add cordova-plugin-inappbrowser
 npx webpack
 
 if [ "$MOBILE_PLATFORM" = "ios" ]; then
-  cordova platform add ios || true
-  cordova build ios \
-    --buildFlag="-UseModernBuildSystem=0" \
-    --device \
-    --codeSignIdentity="iPhone Developer" \
-    --developmentTeam="$DEVELOPMENT_TEAM" \
-    --packageType="development" \
-    --buildFlag="-allowProvisioningUpdates"
+  fastlane build
```

Comment:
I would use fastlane also for android if it is easy to implement, mainly for creating less confusion

Reply:
ok, will create a separate PR for that

Reply:
Looking at it, it is quite easy to implement, but brings another unnecessary prerequisite (fastlane) for anybody who will want to run the tests locally. (Also we would need to install it in our pipeline in the build container.) I would prefer not to add this prerequisite and keep it simpler. @b1zzu wdyt?

Reply:
Yes, it makes sense, I want also to keep things simple